library/core/src/cell.rs

```diff
@@ -2246,6 +2246,7 @@ impl<T: CoerceUnsized<U>, U> CoerceUnsized<SyncUnsafeCell<U>> for SyncUnsafeCell
 impl<T: DispatchFromDyn<U>, U> DispatchFromDyn<SyncUnsafeCell<U>> for SyncUnsafeCell<T> {}
 
 #[allow(unused)]
+#[allow(useless_send_constraint)]
```

Comment:
This line currently causes compilation to fail as it is unrecognised lint

Reply:
Gating the attribute on stage>=1 with `bootstrap` cfg should do it:
```suggestion
#[cfg_attr(not(bootstrap), allow(useless_send_constraint))]
```